ncm-spma/src/main/pan/components/spma/schema.pan

```diff
@@ -90,6 +93,6 @@ type component_spma_type = {
 };
 
 bind "/software/components/spma" = component_spma_type;
-bind "/software/groups" = SOFTWARE_GROUP{};
+bind "/software/groups" = SOFTWARE_GROUP;
```

Comment:
Yes. We decided to not to support optional/default/mandatory group sets as RH tests only with the default set and deps may fail/have interesting consequences for other setups.